src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3019,6 +3020,15 @@ SynthesizedRecordConstructor addCtor(RecordDeclarationSyntax declWithParameters)
                 return ctor;
             }
 
+            void addDeconstructor(ImmutableArray<ParameterSymbol> parameters, ImmutableArray<PropertySymbol> properties)
```

Comment:
Minor: Perhaps name this `addDeconstruct`. (I've reviewed this change a couple of times, each time thinking this refers to a destructor initially.)

Reply:
Done